Packs/IBMResilientSystems/Integrations/IBMResilientSystems/IBMResilientSystems.py

```diff
@@ -1029,15 +1025,13 @@ def update_table_command(client, args):
     elif isinstance(args['row'], dict):
```

Comment:
from what I know the argument should always be coming in as a string. Do you see these two blocks for list/dict get run occasionally?